schemas/com.endlessm.panel-extension.gschema.xml

```diff
@@ -1,5 +1,49 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <schemalist>
   <schema path="/com/endlessm/panel-extension/" id="com.endlessm.panel-extension" >
+    <key name="hot-corner-enabled" type="b">
+      <default>false</default>
+      <summary>
+        Enables the hot corner for the window picker
+      </summary>
+      <description>
+        By default, the hot corner for the window picker is disabled.
+        Set to true to enable the hot corner.
+      </description>
+    </key>
+    <key name="hot-corner-on-right" type="b">
+      <default>true</default>
+      <summary>
+        Set the left/right position of the hot corner for the window picker
+      </summary>
+      <description>
+        Default position of the hot corner is in a right corner.
+        Set false to move the hot corner to a left corner.
+        The position is not affected by the RTL vs. LTR text direction.
+      </description>
+    </key>
+    <key name="hot-corner-on-bottom" type="b">
+      <default>true</default>
+      <summary>
+        Set the top/bottom position of the hot corner for the window picker
+      </summary>
+      <description>
+        Default position of the hot corner is in a bottom corner.
+        Set false to move the hot corner to a top corner.
+      </description>
+    </key>
```

Comment:
This might be a good chance to get rid of this messy composition. It's probably better to introduce an uint `hot-corner-position` key, which would act as a flag (top/down being the first bit, left/right being the second bit)

Reply:
Yes, that could simplify the settings but not sure if it would make it harder for users - I would prefer to not change this now tbh but let me know.

Reply:
Which users? The people who are able to reach these GSettings are very likely also able to read the description of the setting (which would list the 4 values here). I don't think this is a reason to introduce convoluted code, or an argument whatsoever.

The reason I think it is a good idea to do it now is that migration didn't happen yet; once this lands, and is released, there's no turning back.

Reply:
Ok, maybe users is not a good argument, but the other argument is that I don't see how having a hot-corner-position instead of 2 keys that are more clear would help making the code less convulated (note the code wouldn't change much, it would basically change to the keys are read and add some complexity to the migration).

Reply:
Lets keep these settings separate for now, if by any chance we want to change this later on we can introduce migration version as you did in the desktop extension.

Reply:
Not changed as explained above.